README.md

```diff
@@ -1,6 +1,13 @@
 # Airship
 > Airship is a components library built by CARTO
 
+[![CircleCI](https://circleci.com/gh/CartoDB/airship/tree/master.svg?style=svg)](https://circleci.com/gh/CartoDB/airship/tree/master)
+
+There are two npm packages, for easier reuse:
+* @carto/airship-components [![npm version](https://badge.fury.io/js/%40carto%2Fairship-components.svg)](https://badge.fury.io/js/%40carto%2Fairship-components)
```

Comment:
i would remove the name and let only the badge

Reply:
Regarding to badges, that's fine for me.
I guess we'll have to explain this "split" with more detail elsewhere (DEVELOPERS.md, future docs...), but we'll leave that for future PRs